definitions/materialized/neftyquest_leaderboard_template.sql

```diff
@@ -0,0 +1,75 @@
+CREATE MATERIALIZED VIEW IF NOT EXISTS nefy_quest_leaderboard_{{quest_id}} AS
+SELECT rank() OVER (ORDER BY experience DESC) "rank",
+       account,
+       total_sold,
+       total_bought,
+       items_sold,
+       items_bought,
+       (total_collected / {{total_to_collect}}::numeric) completion_percentage,
+       experience,
+       tokens.token_symbol symbol,
+       tokens.token_precision symbol_precision
+FROM (
+     SELECT
+       COALESCE(seller, buyer) account,
+        COALESCE(nefty_sells.total_sold, 0) total_sold,
+        COALESCE(nefty_buys.total_bought, 0) total_bought,
+        COALESCE(nefty_sells.sold_items, 0) items_sold,
+        COALESCE(nefty_buys.bought_items, 0) items_bought,
+        COALESCE(templates_owned.total, 0) total_collected,
+        COALESCE(nefty_sells.symbol, nefty_buys.symbol) AS symbol,
+        (
+                (CASE COALESCE(templates_owned.total, 0) WHEN {{total_to_collect}} THEN {{completion_multiplier}} ELSE 1 END) * 100 *
+                (
+                        (COALESCE(nefty_sells.total_sold, 0) + COALESCE(nefty_buys.total_bought, 0)) / {{volume_threshold}} +
+                        (COALESCE(nefty_sells.sold_items, 0) + COALESCE(nefty_buys.bought_items, 0)) * {{points_per_asset}}
+                )
+        ) experience
+    FROM
+    (
+        SELECT seller, count(*) FILTER (WHERE final_price > {{min_asset_value}}) as sold_items,
+        SUM(final_price)                               as total_sold,
+        settlement_symbol                              as symbol
+        FROM atomicmarket_sales
+        WHERE maker_marketplace = '{{marketplace}}'
+        AND state = {{state}}
+        AND updated_at_time
+            > '{{start_time}}'
+        AND updated_at_time
+            < '{{end_time}}'
+        GROUP BY seller, settlement_symbol
+    ) AS nefty_sells
+    FULL OUTER JOIN
+    (
+        SELECT buyer,
+        count(*) FILTER (WHERE final_price > {{min_asset_value}}) as bought_items,
+        SUM(final_price)                               as total_bought,
+        settlement_symbol                              as symbol
+        FROM atomicmarket_sales
+        WHERE taker_marketplace = '{{marketplace}}'
+        AND state = {{state}}
+        AND updated_at_time
+            > '{{start_time}}'
+        AND updated_at_time
+            < '{{end_time}}'
+        GROUP BY buyer, settlement_symbol
+    ) AS nefty_buys
+    ON nefty_buys.buyer = nefty_sells.seller
+    FULL OUTER JOIN
+    (
+        SELECT owner, COUNT(DISTINCT template_id) total
+        FROM atomicassets_assets
+        WHERE template_id IN ({{templates}})
+        AND atomicassets_assets.updated_at_time < '{{end_time}}'
```

Comment:
maybe we should filter the owner is not null here, I assume it comes null when the asset has been burned so it should be irrelevant for this list

Reply:
I added it